ui/analyse/src/study/studyChapters.ts

```diff
@@ -134,28 +166,24 @@ export function view(ctrl: StudyCtrl): VNode {
         },
       },
     },
-    ctrl.chapters
-      .list()
-      .map((chapter, i) => {
-        const editing = ctrl.chapters.editForm.isEditing(chapter.id),
-          loading = ctrl.vm.loading && chapter.id === ctrl.vm.nextChapterId,
-          active = !ctrl.vm.loading && current && !ctrl.relay?.tourShow.active && current.id === chapter.id;
-        return h(
-          'div',
-          {
-            key: chapter.id,
-            attrs: { 'data-id': chapter.id },
-            class: { active, editing, loading, draggable: canContribute },
-          },
-          [
-            h('span', loading ? h('span.ddloader') : ['' + (i + 1)]),
-            h('h3', chapter.name),
-            chapter.ongoing ? h('ongoing', { attrs: { ...dataIcon(''), title: 'Ongoing' } }) : null,
-            !chapter.ongoing && chapter.res ? h('res', chapter.res) : null,
-            canContribute ? h('i.act', { attrs: dataIcon('') }) : null,
-          ]
-        );
-      })
+    [
+      ctrl.chapters.list().length > 7
+        ? h('input.chapter-filter', {
+            on: { input: filterChapters },
+            attrs: { value: filterChInputValue, label: 'filter chapters', placeholder: 'filter chapters' },
```

Comment:
I don't think there's a need to set the `value` here? Also, `Filter chapters` should probably be capitalized.